doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -479,19 +479,21 @@ toplevel. This kind of file is called a *script* for Coq. The standard
 (and default) extension of Coq’s script files is .v.
 
 
-.. cmd:: Load {? Verbose } {| @string | @ident }
+.. cmd:: Load {? Verbose } {| @string | @qualid }
 
-   Loads a file.  If :n:`@ident` is specified, the command loads a file
-   named :n:`@ident.v`, searching successively in
-   each of the directories specified in the *loadpath*. (see Section
-   :ref:`libraries-and-filesystem`)
+   Executes the commands of a file in the current context.
```

Comment:
```suggestion
   Executes the commands of a :n:`.v` file in the current context.
```

Is that right--only .v files can be loaded?  If not, is it really useful to load non-.v files?

Reply:
The contents of the file must always be something that's valid for a .v file.  The ability to load files with different extensions might be useful?

Reply:
In particular, if you can Load .out files, then you can combine `Redirect` with `Load` to dynamically emit Gallina (e.g., via Ltac `idtac`) and then run it.

Reply:
Gotta be careful about "might be useful".
FWIW, in coqtop, Redirect silently overwrites any existing file.  It doesn't do an append.
In CoqIDE, Redirect creates an empty file and prints the idtac in the Messages panel.